views/templates/admin/configure.tpl

```diff
@@ -36,7 +36,6 @@
     {else}
     <li class="active"><a href="#data_feed_tab" role="tab" data-toggle="tab">{l s='Data Feed' mod='doofinder'}</a></li>
     {if !$dfEnabledV9 && !$is_new_shop}
-        <li><a href="#internal_search_tab" role="tab" data-toggle="tab">{l s='Internal Search' mod='doofinder'}</a></li>
         <li><a href="#custom_css_tab" role="tab" data-toggle="tab">{l s='Custom CSS' mod='doofinder'}</a></li>
```

Comment:
Also delete this tab. it's useless

Reply:
Removed

Reply:
All v7 related settings have been removed. If a user with v7 installs the latest version of the plugin, they will still be able to use the v7 layer, but will not have access to any related settings. The settings you have when you update are kept.

Reply:
@ivan-doofinder  what do we do? Do we remove the custom css tab too or do we keep it?

Reply:
The tab currently doesn't work, so it doesn't make sense for it to be there.